testcases/kernel/syscalls/utimensat/utimensat01.c

```
@@ -152,8 +160,9 @@ int main(int argc, char *argv[])
 
 	if (dirfdPath != NULL) {
 		dirfd = open(dirfdPath, oflag);
+		/* Ignoring open error to test EBADF scenario */
```

What? Ignore the error why?

There's a change here from the original, why?